web/babel.config.cjs

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Is Babel for `jest`?

Reply:
Yes, it is used for transpiling TypeScript for jest and will be also used for precompiling .svelte files so Svelte components can be tested.

Reply:
Does it get in the way of Vite and the development flow, or is it only transpiling on the `test` command?

Reply:
No it's only used by Jest, since it doesn't understand `.ts` or `.svelte` babel is needed to transpile and precompile to JavaScript before running the tests 😃

Reply:
@jbaez jest also a plugin for typescript instead of babel, have you ever tried it?

Reply:
> @jbaez jest also a plugin for typescript instead of babel, have you ever tried it?

`ts-jest` yes I have. I have used both `Babel` and `ts-jest` in different projects and personally would be inclined to use Babel. The reason is that I prefer transpilation only when doing TDD, gives faster feedback without worrying yet about without type checking, unused arguments wrong return type, etc. There would be a `check:all` NPM command like in the server that would also run TS type checking.

In this case we need to use Babel anyway for Svelte precompilation in jest, so that makes it an even easier choice to use it also for TypeScript.